packages/client/src/interfaces.ts

```diff
@@ -98,4 +98,6 @@ export interface IArchiveResponseBody {
         final: boolean
         blockHeight: number
     }
+    startDate: number
```

Comment:
Same as above, shouldn't the date be handled as a `string` or returned as a `Date` object?